qa/tasks/cephfs/mount.py

```diff
@@ -240,7 +240,7 @@ def _verify(self, proc, retval=None, errmsg=None):
             assert proc.returncode == retval, msg
 
         if errmsg:
-            stderr = proc.stderr.getvalue().lower()
+            stderr = proc.stderr.getvalue().decode().lower()
```

Comment:
why not `six.ensure_str` here?

Reply:
> ah yeah, it is a unicode, still needs to use `six.ensure_str` here

could you be more specific? what is a unicode. and why do we still need to use `six.ensure_str` here?